aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -27,6 +25,9 @@ module AptosFramework::Genesis {
         initial_version: u64,
         consensus_config: vector<u8>,
         min_price_per_gas_unit: u64,
+        epoch_interval: u64,
+        minimum_stake: u64,
+        maximum_stake: u64,
```

Comment:
feels like there shouldn't be a maximum stake defined but more inferred based upon available stake?

Reply:
this is used to limit the centralization, but good point it has to be relative to the total stake too